lib/proforma/helpers/import_helpers.rb

```diff
@@ -74,13 +74,17 @@ def test_files_from_test_configuration(test_configuration_node)
         files_from_filerefs(test_configuration_node.search('filerefs'))
       end
 
-      def any_data_tag(any_data_node)
-        [].tap do |any_data|
-          return any_data if any_data_node.nil?
-
-          any_data_node.children.each do |any_data_tag|
-            any_data << {namespace: any_data_node.children.first.namespace.prefix, key: any_data_tag.name,
-                         value: any_data_tag.children.first.text}
+      def meta_data(any_data_node, use_namespace: false)
+        # use_namespace forces the use of the namespace as hash key - it should only be used at the entry of the recursion
+        {}.tap do |any_data|
+          any_data_node.children.each do |node|
+            key = (use_namespace ? node.namespace.prefix : any_data_node.name).to_sym
+            any_data[key.to_sym] = if node.node_type == Nokogiri::XML::Node::TEXT_NODE
```

Comment:
Is it required to call the method `to_sym` here again? I would have expected that it is sufficient to have the call `to_sym` once in line 81?

Reply:
you are correct - fixed!